web/src/message_edit.ts

```diff
@@ -659,6 +659,8 @@ function edit_message($row: JQuery, raw_content: string): void {
         create_copy_to_clipboard_handler($row, the($copy_message), $message_edit_content);
     } else {
         $copy_message.remove();
+        // Initialise the compose tooltips for the edit box.
+        compose_tooltips.initialize_compose_tooltips();
```

Comment:
@shuklamaneesh23  this call is a duplicate of the next one a little further down. Please rebase the commits to remove this one.